### PR TITLE
Miscellaenous doc fixes

### PR DESCRIPTION
**Pull request by dstrain115**
Last round of small doc fixes to get in.

- Add more extensive DocString for Device.
- Fixed parens on swap gate
- Another try to fix broken example text on ArithmeticGate
- Miscellaenous commas missing
- Fix Note on azure-quantum page
- Fix some code text on TaggedOperations

### CODE REVIEW

File: cirq-core/cirq/qis/measures.py [hunk 1]
```diff
@@ -257,7 +259,10 @@ def von_neumann_entropy(
     validate: bool = True,
     atol: float = 1e-7,
 ) -> float:
-    """Calculates the von Neumann entropy of a quantum state in bits.
+    r"""Calculates the von Neumann entropy of a quantum state in bits.
```
Comment (viathor) [inline cirq-core/cirq/qis/measures.py:262]: Optional: This is sometimes specialized to "ebits" (bits of bipartite entanglement).

File: cirq-core/cirq/devices/device.py [hunk 2]
```diff
@@ -22,7 +22,37 @@
...
-    """Hardware constraints for validating circuits."""
+    """Hardware constraints for validating circuits.
+
+    This class is an interface for representing constraints and
+    structures of quantum hardware devices.
+
+    This interface is split into two parts: validation and
+    exploration.  The primary responsibility of this class
+    is to validate circuits (ie. can this device execute the
+    circuit as-is?).  The secondary responsibiliy of the class
```
Comment (95-martin-orion) [inline cirq-core/cirq/devices/device.py:33]: `responsibiliy` -> `responsibility`

**Review by viathor — APPROVED** [review]

File: docs/hardware/azure-quantum/access.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
-> *) Note that there are two packages that can be used to access IonQ devices with `cirq`: `azure-quantum` or `cirq-ionq`. If you would like to access IonQ via a new or existing Azure subscription, use `azure-quantum` by following the steps in this tutorial. If you already have a preexisting IonQ API key through other means, follow the steps outlined in [Cirq with the IonQ API](/cirq/hardware/ionq/access.md) to use `cirq-ionq` instead.
+Note: that there are two packages that can be used to access IonQ devices with `cirq`: `azure-quantum` or `cirq-ionq`. If you would like to access IonQ via a new or existing Azure subscription, use `azure-quantum` by following the steps in this tutorial. If you already have a preexisting IonQ API key through other means, follow the steps outlined in [Cirq with the IonQ API](/cirq/hardware/ionq/access.md) to use `cirq-ionq` instead.
```
Comment (95-martin-orion) [inline docs/hardware/azure-quantum/access.md:5]: `Note: that there` -> `Note: there`

**Review by 95-martin-orion — APPROVED** [review]
nit nit

File: cirq-core/cirq/qis/measures.py (hunk 1 above)
Comment (dstrain115) [reply cirq-core/cirq/qis/measures.py:262]: Added to description.

File: cirq-core/cirq/devices/device.py (hunk 2 above)
Comment (dstrain115) [reply cirq-core/cirq/devices/device.py]: Done.

File: docs/hardware/azure-quantum/access.md (hunk 3 above)
Comment (dstrain115) [reply docs/hardware/azure-quantum/access.md]: Done.

**Pull request merged**